packages/manager/src/views/SPAPropertyDetailPage/VirtualPath/components/CreateVirtualPath.tsx

```diff
@@ -0,0 +1,270 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Controller, useForm } from 'react-hook-form';
+import * as yup from 'yup';
+
+import { useGetWebPropertyGroupedByEnv } from '@app/services/persistent';
+import {
+  ActionGroup,
+  Button,
+  Form,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  Split,
+  SplitItem,
+  TextInput,
+  Tooltip
+} from '@patternfly/react-core';
+
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+const envValidation = /^[a-zA-Z0-9-]+$/;
+
+export const schema = yup.object({
+  propertyIdentifier: yup.string().label('Web property').trim(),
+  identifier: yup.string().label('Application Name').trim(),
+  virtualPath: yup
+    .string()
+    .label('Source File Path')
+    .trim()
+    .required()
+    .test(
+      'no-leading-or-trailing-slashes',
+      'Source File Path should not start or end with a slash',
```

Comment:
Please remove this

Reply:
We need this right for the path?

Reply:
You can change it to base path or virtual path, as it's not the file path